docs/schema/clientwebpartproperty-complextype-apphostwebfeatures.md

```diff
@@ -18,11 +19,9 @@ ms.assetid: deebfdda-704b-2393-fdb7-8b6292836a0e
 
 ## Type information
 
-|   |   |
-|---|---|
-| **Namespace**  | `http://schemas.microsoft.com/sharepoint/` |
-| **Schema file**  | apphostwebfeatures.xsd |
-| **Extension base**  | None |
+| Namespace | Schema File | Extension Base |
```

Comment:
Please turn this into a 2 table column with suggested headers of "Name/Value" or "Information/Location" or whatever you feel is most appropriate.